geograypher/cameras/cameras.py

```diff
@@ -15,13 +15,19 @@
 from pyvista import demos
 from scipy.spatial.distance import pdist
 from shapely import Point
+from shapely.geometry import Multipolygon, Polygon
```

Comment:
You can directly import from shapely, as done above. Not important, it would just make it cleaner.